cmd/cluster-agent/subcommands/config/command.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows && kubeapiserver
+// +build !windows,kubeapiserver
+
+// Package config implements 'cluster-agent config'.
+package config
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/cluster-agent/command"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/cli/subcommands/config"
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/config/settings"
+	settingshttp "github.com/DataDog/datadog-agent/pkg/config/settings/http"
+
+	"github.com/spf13/cobra"
+)
+
+const loggerName = "CLUSTER"
```

Comment:
loggerName seems to be defined in several places, can we create a dedicated package for const values? and reuse the same const variable everywhere?